roles/vault_pki/README.md

```diff
@@ -47,4 +47,5 @@ Role variables
     * `vault_pki_write_certificates_host:` The host on which certificates will be written to. (default: "localhost")
     * `vault_pki_certificates_directory`: directory to output certificate files to.
     * `vault_pki_write_certificates`: whether to write generated certificates to a file or not (default: `false`)
+    * `vault_pki_write_pem_bundle`: output the generated certificate and private key into a single file (default: `false`)
```

Comment:
Default of false is a change in behaviour. Is that what we want?

Reply:
Most things don't use pem bundles. They use a cert and key. So I'd say yes we want this change in behaviour.